When running the tests locally on Ubuntu I'm getting errors in `test_graph__rmw_zenoh_cpp` and `test_services__rmw_zenoh_cpp`

 - `test_graph__rmw_zenoh_cpp`
```bash
98: /home/ahcorde/ros2_rolling/src/ros2/rcl/rcl/test/rcl/test_graph.cpp:1078: Failure
98: Expected: (attempts) <= (max_attempts), actual: 101 vs 100
98: Unable to attain all required nodes
```

 - `test_services__rmw_zenoh_cpp`

```bash
121: [service_fixture-1] [ERROR] [1742575407.925265153] [rcl]: Service never became ready
121: [client_fixture-2] [ERROR] [1742575407.932023887] [rcl]: Server never became available
```

--- a/rcl/test/CMakeLists.txt
+++ b/rcl/test/CMakeLists.txt
@@ -182,11 +182,10 @@ function(test_target)
   set(rmw_implementation_env_var RMW_IMPLEMENTATION=${rmw_implementation})
 
   # If the rmw_implementation is rmw_zenoh_cpp, run the tests with multicast discovery enabled.
+  # Note: This is a temporary change that will be reverted before we branch into Kilted.
   if(rmw_implementation STREQUAL "rmw_zenoh_cpp")
     list(APPEND rmw_implementation_env_var
-      ZENOH_ROUTER_CHECK_ATTEMPTS=-1
       ZENOH_CONFIG_OVERRIDE=scouting/multicast/enabled=true
-      RUST_LOG=z=error
     )
   endif()
 

--- a/rcl_action/CMakeLists.txt
+++ b/rcl_action/CMakeLists.txt
@@ -137,11 +137,10 @@ if(BUILD_TESTING)
     set(rmw_implementation_env_var RMW_IMPLEMENTATION=${rmw_implementation})
 
     # If the rmw_implementation is rmw_zenoh_cpp, run the tests with multicast discovery enabled.
+    # Note: This is a temporary change that will be reverted before we branch into Kilted.
     if(rmw_implementation STREQUAL "rmw_zenoh_cpp")
       list(APPEND rmw_implementation_env_var
-        ZENOH_ROUTER_CHECK_ATTEMPTS=-1
         ZENOH_CONFIG_OVERRIDE=scouting/multicast/enabled=true
-        RUST_LOG=z=error
       )
     endif()
 

